### PR TITLE
Fix markdown and remove non-English characters

### PR DESCRIPTION
**Pull request by ruromero**
Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

* Replace non-English characters
* Add alt text for diagrams
* Remove trailing `#` for headers
* There should be only one H1 (# Title)
* Wrap code and use spaces instead of tabs
* Remove extra spaces and line breaks

### CODE REVIEW

File: workflow/spec/spec.md [hunk 1]
```diff
@@ -345,89 +334,94 @@ Workflow.
...
+    "source": "CloudEvent source UUID",
+    "correlation-token": "A path string to an identity label field in the event message"
+    }
+```
 
-The "EVENT-NAME” is used by the EVENTS-EXPRESSION in an Event state.
-Note that the event itself is defined according to the [CloudEvents Specification]
-(https://cloudevents.io/). The workflow only has a reference to the event name.
+The "EVENT-NAME" is used by the EVENTS-EXPRESSION in an Event state.
+Note that the event itself is defined according to the [CloudEvents Specification](https://cloudevents.io/). The workflow only has a reference to the event name.
```
Comment (duglin) [inline workflow/spec/spec.md]: Can you wrap this at 80 cols?

File: workflow/spec/spec.md [hunk 2]
```diff
@@ -469,188 +466,180 @@ the event state.
...
-The field “timeout” specifies the time period waiting for the events in the
+The field "timeout" specifies the time period waiting for the events in the
 EVENTS-EXPRESSION. If the events do not come within the timeout period, the
 workflow will transit to end state.
 
 The field "action-mode" specifies whether functions are executed in sequence or
 in parallel and could either be SEQUENTIAL or PARALLEL.
 
-The "actions" field is a list of **ACTION-DEFINITION** constructs that specify
+The "actions" field is a list of [ACTION-DEFINITION](#action-definition-(action-definition)) constructs that specify
```
Comment (duglin) [inline workflow/spec/spec.md]: wrap at 80

**Comment by duglin** [comment]
@ruromero thanks a ton!  If you can just fix the lines that need to be wrapped at 80, I'll merge it

**Comment by ruromero** [comment]
> @ruromero thanks a ton! If you can just fix the lines that need to be wrapped at 80, I'll merge it

You are more than welcome. Happy to contribute.

**Comment by duglin** [comment]
thanks!!

**Pull request merged**